### DISCUSSION TITLE
Install failing "systemd-run" failed with return code: 1

### DISCUSSION THREAD

**Discussion by Pandamonium14** [Security-Onion-Solutions/securityonion#5680] [Unsupported Versions]
I'm attempting to install SO2 on a VM in virtualbox, but I keep running into errors during install. Initially after install mysql wasn't running so I forced it to start with `sudo so-mysql-start`. All services are running as ok in status, but when I ran `sudo grep -E "ERROR|Result: False" /root/sosetup.log` I'm getting the below install error response. 

![siem install error](https://user-images.githubusercontent.com/91503556/135005972-95696920-9cf8-4cda-bfd1-df5eae700770.JPG)

I'm not 100% sure what's it's pointing to as the failure point beyong the 'pkg.clean_metadata. And I've got no idea how to fix it.

**Comment by dougburks** [comment]
What are the specs of your VM?
